src/pages/en/guides/typescript.md

```diff
@@ -64,7 +64,7 @@ import Layout from '@layouts/Layout.astro';
 }
 ```
 
-## Component Props
+## Component `Props`
```

Comment:
I wouldn’t include this one. `props` lowercase is the code version, here this is just referring to the general concept of “props” rather than a specific variable named “props”.

```suggestion
## Component Props
```

Reply:
Yeah, I was not sure about this one, thanks for the clarification 👍